.github/workflows/cd.yml

```diff
@@ -11,14 +11,25 @@ jobs:
 
     steps:
 
-    - name: Set up JDK 11
+    - name: Set up JDK 14
       uses: actions/setup-java@v1
       with:
-        java-version: 11.0.x
+        java-version: 14.0.x
         server-id: sonatype-nexus-staging
```

Comment:
Is this still necessary?

Reply:
We have so far failed to start using JDK 11 in OpenSearch core, but you are free to do so here.

Reply:
> Is this still necessary?

Not necessary. Removed.

Reply:
> We have so far failed to start using JDK 11 in OpenSearch core, but you are free to do so here.

This is upgrade from JDK 11 to JDK 14 here.

Reply:
I do not refer to JDK change from 11 to 14 that we did for opendistro branches as well (see #1226) and that was supposed to be done as part of #1131 (@cliu123, @debjanibnrj @palashhedau) . My comment applies to `server-id`.